### DISCUSSION TITLE
Integration to WebFlux applications

### DISCUSSION THREAD

**Discussion by ceremo** [LogNet/grpc-spring-boot-starter#400] [General]
Hi,

I was interested in using this project because of the integration of gRPC services to Spring Security, but it seems designed only for Spring MVC applications, is in the future roadmap of the project to give support also to WebFlux applications?

Thanks for your time.

**Comment by jvmlet** [comment]
The security implementation follows the mvc API, but works as grpc server side interceptor. It works as is for everything that run after the security interceptor, including webflux integration
